src/Microsoft.AspNet.Antiforgery/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -84,46 +84,46 @@ public AntiforgeryToken GenerateCookieToken()
                         nameof(DefaultAntiforgeryAdditionalDataProvider)));
             }
 
-            return formToken;
+            return requestToken;
         }
 
         public bool IsCookieTokenValid(AntiforgeryToken cookieToken)
         {
-            return (cookieToken != null && cookieToken.IsSessionToken);
+            return (cookieToken != null && cookieToken.IsCookieToken);
         }
 
         public void ValidateTokens(
             HttpContext httpContext,
-            AntiforgeryToken sessionToken,
-            AntiforgeryToken fieldToken)
+            AntiforgeryToken cookieToken,
+            AntiforgeryToken requestToken)
         {
             if (httpContext == null)
             {
                 throw new ArgumentNullException(nameof(httpContext));
             }
 
-            if (sessionToken == null)
+            if (cookieToken == null)
             {
                 throw new ArgumentNullException(
-                    nameof(sessionToken),
+                    nameof(cookieToken),
                     Resources.Antiforgery_CookieToken_MustBeProvided_Generic);
             }
 
-            if (fieldToken == null)
+            if (requestToken == null)
             {
                 throw new ArgumentNullException(
-                    nameof(fieldToken),
-                    Resources.Antiforgery_FormToken_MustBeProvided_Generic);
+                    nameof(requestToken),
+                    Resources.Antiforgery_RequestToken_MustBeProvided_Generic);
             }
 
             // Do the tokens have the correct format?
-            if (!sessionToken.IsSessionToken || fieldToken.IsSessionToken)
+            if (!cookieToken.IsCookieToken || requestToken.IsCookieToken)
             {
                 throw new InvalidOperationException(Resources.AntiforgeryToken_TokensSwapped);
             }
 
             // Are the security tokens embedded in each incoming token identical?
-            if (!Equals(sessionToken.SecurityToken, fieldToken.SecurityToken))
+            if (!Equals(cookieToken.SecurityToken, requestToken.SecurityToken))
```

Comment:
Could we rewrite this as `cookieToken.SecurityToken.Equals(requestToken.SecurityToken)`? Feel free to leave as is if you think it doesn't make it more readable.